[16.0][IMP] sale_global_discount: refactor _compute_amounts

trisdoan opened the pull request:
### Context

In simple case (tax is not included in price), flow of the module as follow:
   - Let tax is computed as usual, as odoo native supports
   - Afterwards, checking if global discount is applied on Sale order:
      1. If not, only add amt_untaxed_before_global_discount and amt_total_before_global_discount,
      2. If yes, precompute tax

**A example of Sale order is below:**
- Tax is not included in price
- With 20% global discount

<div style="display: flex;">

<div style="flex: 1;">

| **Product Name** | **Quantity** | **Unit Price** | **SubTotal** |
|------------------|--------------|----------------|-----------------|
| Example Product  | 2            | $75        | $150        |

</div>

<div style="flex: 0; margin-left: 20px;">

| **Untaxed Amt** |
|------------------|
| $120      |

| **Tax** |
|------------------|
| $18.0      |

| **Total** |
|------------------|
| $138.0      |

**=> So tax is recalculated based on Final price (after global discount)**


- Before this change, if enabling included-in-price tax, it breaks the intended flow as it triggers the part 'handling price include' again with the new price

### This change
- pass param: `handle_price_include=False` because it was handle in super(), by default `handle_price_include=True` . Now, as we recalculate tax to apply global discount, there is no need to handle price included or not.

- With above example, now it is:
   **- but this time, Tax is included in price**

<div style="display: flex;">

<div style="flex: 1;">

| **Product Name** | **Quantity** | **Unit Price** | **SubTotal** |
|------------------|--------------|----------------|-----------------|
| Example Product  | 2            | $75        | $130.43       |

</div>

<div style="flex: 0; margin-left: 20px;">

| **Untaxed Amt** |
|------------------|
| $104.34      |

| **Tax** |
|------------------|
| $15.65      |

| **Total** |
|------------------|
| $119.9900      |


- Add a check to only recompute tax when global discount is applied on the sale order. Before this change, it results in rounding issue. It happens when products are configured as tax-included in price. For example:

![sample_sale_global_discount](https://github.com/trisdoan/sale-workflow/assets/88806544/513d414d-c0d3-49ad-ad53-17fe66731792)

CarlosRoca13 committed:
[ADD] sale_order_product_assortment: New module to use the product assortment
This module allow to use the product assortment related to partners on sale orders.

CarlosRoca13 committed:
[IMP] sale_order_product_assortment: black, isort, prettier

CarlosRoca13 committed:
[MIG] sale_order_product_assortment: Migration to v13

danimv5 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (9 of 9 strings)

Translation: sale-workflow-13.0/sale-workflow-13.0-sale_order_product_assortment
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-13-0/sale-workflow-13-0-sale_order_product_assortment/ca/

sergio-teruel committed:
[IMP] sale_order_product_assortment: Improve domain

Use product_assortment domain and only one field to get allowed products result from whitelist and blacklist

TT30584

sergio-teruel committed:
[FIX] sale_order_product_assortment: Make invisible column product_allowed_id
TT33641

sergio-teruel committed:
[IMP] product_assortment: Add blacklist product domain to exclude some products
TT36814

ilyasProgrammer committed:
[MIG] sale_order_product_assortment: Migration to 14.0

francesco-ooops committed:
Added translation using Weblate (Italian)

CarlosRoca13 committed:
[IMP] sale_order_product_assortment: black, isort, prettier

CarlosRoca13 committed:
[MIG] sale_order_product_assortment: Migration to version 15.0

CarlosRoca13 committed:
[ADD] sale_planner_calendar: New module to manage commercial visits

sergio-teruel committed:
[FIX] sale_planner_calendar: Access to actions

CarlosRoca13 committed:
[IMP] sale_planner_calendar: Add customers to calendar planner view

dessanhemrayev committed:
sale_order_partner_no_autofollow:Expands the sale module, disables adding a partner to autofollow

carlosdauden committed:
[FIX] sale_planner_calendar: Use same domain in Sales Follower Orders and Sales Follower Order Lines
TT40792

OCA-git-bot committed:
sale_planner_calendar 15.0.1.1.0

Commit:
Update sale_order_partner_no_autofollow.pot

OCA-git-bot committed:
icon.png

OCA-git-bot committed:
README.rst

carlosdauden committed:
[IMP] sale_order_product_assortment: Use original product_id field keeping domain

carlosdauden committed:
[IMP] sale_order_product_assortment: Increase performance using new get_all_partner_ids cached method

sergio-teruel committed:
[FIX] sale_planner_calendar: Do not send invitation attendees emails when reassign user to calendar.event
TT41889

CarlosRoca13 committed:
[IMP] sale_planner_calendar: link target partners and add domain to user fields

OCA-git-bot committed:
sale_planner_calendar 15.0.1.1.1

sergio-teruel committed:
[FIX] sale planner calendar  Singleton when multi summaries distinct types
TT43294

carlosdauden committed:
[FIX] sale_planner_calendar: Filter button closes wizard if is opened from RedirectWarning popup.
Inline is not possible because v15 try save partner record.
Use of "ir.action.do_nothing" can cause concurrency problems in registry writing.

TT43060

OCA-git-bot committed:
sale_planner_calendar 15.0.1.1.2

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_planner_calendar
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_planner_calendar/

OCA-git-bot committed:
sale_planner_calendar 15.0.1.1.3

sergio-teruel committed:
[FIX] sale_planner_calendar: Event_type_id no exists in planner calendar event
TT43335

OCA-git-bot committed:
sale_planner_calendar 15.0.1.1.4

Commit:
[IMP] sale_planner_calendar: add event type to views TT43459

OCA-git-bot committed:
sale_planner_calendar 15.0.1.2.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_planner_calendar
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_planner_calendar/

carlosdauden committed:
[IMP] sale_order_product_assortment: Use new field applied_assortments_ids

carlosdauden committed:
[IMP] sale_planner_calendar: Allow select action for sale orders

angelmoya committed:
[MIG] sale_planner_calendar: Migration to 16.0

OCA-git-bot committed:
sale_planner_calendar 15.0.1.2.1

CarlosRoca13 committed:
[IMP] sale_planner_calendar: Add button to create sale.order from event tree and kanban view

sergio-teruel committed:
[IMP] sale_planner_calendar: Add settings menu. Make issue types translatable
TT44410

CarlosRoca13 committed:
[FIX] sale_planner_calendar: undefined instead of false on calendar_summary_id

OCA-git-bot committed:
sale_planner_calendar 15.0.1.3.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_planner_calendar
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_planner_calendar/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (3 of 3 strings)

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_order_product_assortment
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_order_product_assortment/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (202 of 202 strings)

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_planner_calendar
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_planner_calendar/es/

CarlosRoca13 committed:
[FIX] sale_planner_calendar: Recurrent events not removed when changing the recurrence from the planner

By doing this change we avoid avoid loose information when accesing
calendar_event_ids of the recurrence_id.

carlosdauden committed:
Added translation using Weblate (Catalan (ca_ES))

OCA-git-bot committed:
sale_planner_calendar 15.0.1.3.1

OCA-git-bot committed:
README.rst

ChristianSantamaria committed:
sale_order_partner_no_autofollow: black, isort, prettier

ChristianSantamaria committed:
sale_order_partner_no_autofollow: fix res.config.settings parameter

s

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (202 of 202 strings)

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_planner_calendar
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_planner_calendar/es/

CarlosRoca13 committed:
[IMP] sale_planner_calendar: Use same method as kanban button to open sale orders

CarlosRoca13 committed:
[FIX] sale_planner_calendar: Remove search method overwrite and set key on domains

ChristianSantamaria committed:
sale_order_partner_no_autofollow: Migration to 15.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_planner_calendar
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_planner_calendar/

josep-tecnativa committed:
[IMP] sale_planner_calendar: test performance improvement
- Switch to setUpClass for avoiding repeat the same setup for each test.
- Include context keys for avoiding mail operations overhead.

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_planner_calendar
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_planner_calendar/

sergio-teruel committed:
[FIX] sale_planner_calendar: Add security group to root menu

CarlosRoca13 committed:
[IMP] sale_planner_calendar: Allow delete recurrence as it can be done from calendar view

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_planner_calendar
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_planner_calendar/

CarlosRoca13 committed:
[FIX] sale_planner_calendar: Avoid infinite loop recursion when recurrence end is forever

sergio-teruel committed:
[IMP] sale_planner_calendar: Do not notify to attendees

sergio-teruel committed:
[IMP] sale_planner_calendar: Do not create attendees

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_planner_calendar
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_planner_calendar/

sergio-teruel committed:
[IMP] sale_planner_calendar: Allow create so from event planned to commercial partner with a system parameter
TT48410

pilarvargas-tecnativa committed:
[IMP] sale_planner_calendar: Design improves and new buttons in calendar planner event

Improvements made:

- Profile for calendar events that will allow an icon to be displayed in the kanban view of the visit. It can also be configured from a recurring event.
- Tree view to edit and add new profiles with their icons which will be fontawesome text strings (e.g fa-user).
- New buttons in the kanban view of the visit: Location, Whatsapp
- Some improvements in the structure to make it responsive with the new addition of buttons that may or may not be visible.

TT48632

jbaudoux committed:
sale_order_carrier_auto_assign: fix all service

EmilioPascual committed:
[ADD] sale_order_line_effective_date

MT-6075 @moduon

sergio-teruel committed:
[FIX] sale_planner_calendar: Do not take into account the user time zone when the planner events are created
TT49169

Commit:
[IMP] sale_global_discount: calculate lines with fixed taxes should be based on product uom qty

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_planner_calendar
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_planner_calendar/

pilarvargas-tecnativa committed:
Translated using Weblate (Spanish)

Currently translated at 97.1% (207 of 213 strings)

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_planner_calendar
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_planner_calendar/es/

pilarvargas-tecnativa committed:
[IMP] sale_planner_calendar: pre-commit stuff

trisdoan committed:
[IMP] sale_global_discount: refactor _compute_amount

trisdoan commented:
Hello @santostelmo, an improved version with test is coming. So this PR can be hold atm, is it ok ?

pilarvargas-tecnativa committed:
[MIG] sale_planner_calendar: continue migration to 16.0

TT44359

OCA-git-bot committed:
Merge PR #3129 into 16.0

Signed-off-by rafaelbn

Commit:
[UPD] Update sale_order_line_effective_date.pot

OCA-git-bot committed:
[BOT] post-merge updates

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 79.5% (35 of 44 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_advance_payment
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_advance_payment/hr/

badbole committed:
Added translation using Weblate (Croatian)

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (7 of 7 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_auto_remove_zero_quantity_lines
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_auto_remove_zero_quantity_lines/hr/

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (3 of 3 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_price_recalculation
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_price_recalculation/hr/

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 85.7% (6 of 7 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_force_invoiced
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_force_invoiced/hr/

badbole committed:
Added translation using Weblate (Croatian)

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (7 of 7 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-product_form_sale_link
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-product_form_sale_link/hr/

badbole committed:
Added translation using Weblate (Croatian)

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (9 of 9 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_delivery_state
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_delivery_state/hr/

badbole committed:
Added translation using Weblate (Croatian)

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (5 of 5 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_mrp_bom
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_mrp_bom/hr/

badbole committed:
Added translation using Weblate (Croatian)

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (7 of 7 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_carrier_auto_assign
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_carrier_auto_assign/hr/

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (4 of 4 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_automatic_workflow_payment_mode
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_automatic_workflow_payment_mode/hr/

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (7 of 7 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_shipping_info_helper
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_shipping_info_helper/hr/

badbole committed:
Added translation using Weblate (Croatian)

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (1 of 1 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_procurement_customer
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_procurement_customer/hr/

badbole committed:
Added translation using Weblate (Croatian)

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (5 of 5 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_discount_display_amount
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_discount_display_amount/hr/

badbole committed:
Added translation using Weblate (Croatian)

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (11 of 11 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_revision
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_revision/hr/

ChristophAbenthungCibex committed:
[FIX]sale_order_revision:fix access error

OCA-git-bot committed:
Merge PR #3088 into 16.0

Signed-off-by rousseldenis

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #3203 into 16.0

Signed-off-by rousseldenis

OCA-git-bot committed:
[BOT] post-merge updates

bealdav committed:
IMP sale_recalculation: allow to recompute several orders

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (24 of 24 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_stock_picking_blocking
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_stock_picking_blocking/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_line_effective_date
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_line_effective_date/it/

Commit:
Added translation using Weblate (Turkish)

Commit:
Translated using Weblate (Turkish)

Currently translated at 100.0% (5 of 5 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_discount_display_amount
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_discount_display_amount/tr/

Commit:
Translated using Weblate (Turkish)

Currently translated at 100.0% (10 of 10 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_line_sequence
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_line_sequence/tr/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (50 of 50 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_block_no_stock
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_block_no_stock/it/

OCA-git-bot committed:
Merge PR #2343 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update sale_global_discount.pot

OCA-git-bot committed:
[BOT] post-merge updates

cyrilmanuel commented:
Could you change the origin of the branch from studio73 to OCA --> Because the original PR was merged already --> https://github.com/OCA/sale-workflow/pull/2343

Shide committed:
[FIX] sale_block_no_stock: Add notes to sale orders when adjusting or moving quantities

Shide committed:
[FIX] sale_order_line_effective_date: Add commmitment date

trisdoan commented:
Hello @cyrilmanuel @santostelmo, I prepared https://github.com/OCA/sale-workflow/pull/3216 to replace this PR

OCA-git-bot committed:
Merge PR #3214 into 16.0

Signed-off-by Shide

Commit:
[UPD] Update sale_block_no_stock.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_block_no_stock
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_block_no_stock/

Commit:
Added translation using Weblate (Turkish)

Commit:
Translated using Weblate (Turkish)

Currently translated at 100.0% (11 of 11 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_revision
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_revision/tr/

OCA-git-bot committed:
Merge PR #3215 into 16.0

Signed-off-by yajo

Commit:
[UPD] Update sale_order_line_effective_date.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_line_effective_date
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_line_effective_date/

OCA-git-bot committed:
Merge PR #3192 into 16.0

Signed-off-by bguillot

Commit:
[UPD] Update sale_order_price_recalculation.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_price_recalculation
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_price_recalculation/

SottomaiorMacedoTec committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (4 of 4 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_price_recalculation
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_price_recalculation/pt_BR/

carolinafernandez-tecnativa committed:
[FIX] sale_planner_calendar: 'calendar.event' object has no attribute 'start_datetime'

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_global_discount
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_global_discount/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (12 of 12 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_global_discount
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_global_discount/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (7 of 7 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_line_effective_date
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_line_effective_date/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (51 of 51 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_block_no_stock
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_block_no_stock/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_price_recalculation
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_price_recalculation/it/

uvid-gordana committed:
Translated using Weblate (Croatian)

Currently translated at 57.8% (11 of 19 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_line_menu
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_line_menu/hr/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (59 of 59 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_automatic_workflow
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_automatic_workflow/it/

carolinafernandez-tecnativa committed:
[IMP] sale_order_product_assortment: pre-commit stuff

carolinafernandez-tecnativa committed:
[MIG] sale_order_product_assortment: Migration to 16.0

TT44355

OCA-git-bot committed:
Merge PR #3229 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update sale_order_product_assortment.pot

OCA-git-bot committed:
[BOT] post-merge updates

dessanhemrayev committed:
[ADD] sale_force_invoiced_quantity: add module

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (3 of 3 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_product_assortment
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_product_assortment/it/

carlos-lopez-tecnativa committed:
[IMP] sale_order_line_sequence: Improved xpath on invoice report
- When installed with other modules, the layout can be inconsistent. This commit makes the xpath more specific.

OCA-git-bot committed:
Merge PR #3239 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #3198 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update sale_planner_calendar.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_planner_calendar
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_planner_calendar/

dalonsofl committed:
[FIX] sale_company_currency: Changes on currency rate application

Borruso committed:
[FIX] sale_blanket_order: fix form view

mymage committed:
Added translation using Weblate (Italian)

OCA-git-bot committed:
Merge PR #3240 into 16.0

Signed-off-by dreispt

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (206 of 206 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_planner_calendar
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_planner_calendar/it/

LauraCForgeFlow committed:
[IMP] sale_delivery_split_date: use security lead time

sbidoul committed:
[MIG] sale_order_partner_no_autofollow from 15 to 16

sbidoul committed:
[IMP] sale_order_partner_no_autofollow: pre-commit auto fixes

sbidoul committed:
[IMP] sale_order_partner_no_autofollow: clarify parameter help text

sbidoul committed:
[MSC] sale_order_partner_no_autofollow: remove maintainers

As requested in https://github.com/OCA/sale-workflow/pull/2715#discussion_r1714367189

OCA-git-bot committed:
Merge PR #3271 into 16.0

Signed-off-by StefanRijnhart

Commit:
[UPD] Update sale_order_partner_no_autofollow.pot

OCA-git-bot committed:
[BOT] post-merge updates

uvid-gordana committed:
Translated using Weblate (Croatian)

Currently translated at 18.6% (11 of 59 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_automatic_workflow
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_automatic_workflow/hr/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (26 of 26 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_product_set
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_product_set/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (170 of 170 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_blanket_order
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_blanket_order/it/

OCA-git-bot committed:
Merge PR #3270 into 16.0

Signed-off-by rousseldenis

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #3233 into 16.0

Signed-off-by rousseldenis

Commit:
[UPD] Update sale_force_invoiced_quantity.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #3241 into 16.0

Signed-off-by StefanRijnhart

OCA-git-bot committed:
[BOT] post-merge updates

rousseldenis committed:
[IMP] sale_automatic_workflow: Remove deprecated queue job context for tests

rousseldenis committed:
[IMP] sale_automatic_workflow: Use of freeze time to test dates

If test is run at a certain date nearly midnight, the test that
evaluates invoice date from sale one was failing. So, freeze the date
at mid of day.

OCA-git-bot committed:
Merge PR #3283 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_partner_no_autofollow
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_partner_no_autofollow/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (6 of 6 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_force_invoiced_quantity
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_force_invoiced_quantity/it/
